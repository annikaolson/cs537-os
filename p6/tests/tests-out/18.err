Traceback (most recent call last):
  File "<string>", line 117, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'd1/d2/d3/d4/d5/d6/d7/d8/d9/file0'
