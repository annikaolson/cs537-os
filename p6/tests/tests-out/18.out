[Errno 28] No space left on device: 'd1/d2'
